### DISCUSSION TITLE
Can anybody help with what's probably a very simple question

### DISCUSSION THREAD

**Discussion by GS868** [swar/Swar-Chia-Plot-Manager#1033] [Q&A]
Hello,

I'm very new to all this and after configuring my config.yaml file i keep getting the following error.

**yaml.scanner.ScannerError: mapping values are not allowed here
  in "C:\Users\Andy\Desktop\Swar-Chia-Plot-Manager-main\config.yaml", line 10, column 8**

I have attached a copy of the file and would greatly appreciate any assistance.
[config yaml.txt](https://github.com/swar/Swar-Chia-Plot-Manager/files/6566568/config.yaml.txt)

Many thanks,

**Comment by FlyingToto** [comment]
not 100% sure but maybe yaml doesn't like spaces in folder names...  try to rename your log folder: **chia-log-files**

```
log:
  # folder_path: This is the folder where your log files for plots will be saved.
  folder_path: C:\Users\Andy\Chia Log Files
```
